source/fundamentals/authentication/mechanisms.txt

```diff
@@ -235,6 +235,22 @@ The driver checks for your credentials in the following sources in order:
       .. literalinclude:: /code-snippets/authentication/aws-env-variable.js
          :language: javascript
 
+.. important:: Retrieval of AWS Credentials
+   
+   Starting in version 4.11, when you install the optional
+   ``aws-sdk/credential-providers`` dependency, the driver uses the AWS SDK
+   to retrieve credentials from the environment. As a result, if you
+   have a shared AWS credentials file or config file, the driver will
+   use those credentials by default if there are no AWS authentication
+   environment variables set.
+   
+   To override this behavior, you can perform one of the following actions:
+   
+   - Set ``AWS_SHARED_CREDENTIALS_FILE`` in your shell or set the
+     equivalent environment variable value in your application.
```

Comment:
Question

Set it to what? I am assuming it is the full path to the credential file, but think it would be helpful to specify this for the reader.